SSDynamicText/NSAttributedString+SSTextSize.h

```diff
@@ -10,6 +10,13 @@
 
 @interface NSAttributedString (SSTextSize)
 
+/**
+ * Create NSAttributedString with each font size changed by delta
+ * @param delta The size by which you want to change the font.
+ * A negative number will lower font.
+ * A positive number will increase font.
+ * @return new NSAttributedString object which font size changed by delta.
```

Comment:
`new NSAttributedString object which font size is changed by delta`
or
`new NSAttributedString object with font size changed by delta`